deploy/cr.yaml

```diff
@@ -194,7 +227,6 @@ spec:
           cpu: "300m"
           memory: "0.5G"
       expose:
-        # enabled: true
```

Comment:
can you get it back? with proper commenting

Reply:
But it doesn't work, if we set it to `false` it just ignores it.
I can put it back like `enabled: true` since that seems to be the default anyway, but in that case we need to fix the option.
Is that what you meant?

Reply:
`expose.enabled` should be disabled by default.
we have it in the config example (`deploy/cr.yaml`) just to simplify the user experience when the user wants to enable it.